blaze/compute/tests/test_url_csv_compute.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+import os
+
+from blaze import Data, compute
+from blaze.utils import raises
+from odo import URL, CSV
+
+import pandas as pd
+import pandas.util.testing as tm
+
+from functools import partial
+
+try:
+    from urllib2 import urlopen
+    from urllib2 import HTTPError, URLError
+except ImportError:
+    from urllib.request import urlopen
+    from urllib.error import HTTPError, URLError
+
+pytestmark = pytest.mark.skipif(raises(URLError,
+                                       partial(urlopen, "http://google.com")),
+                                reason='unable to connect to google.com')
+
+iris_url = ('https://raw.githubusercontent.com/'
```

Comment:
> I would also be fine with something like pytest.skipif(no internet). This looks like a fine way to check for an internet connection.

That's exactly what lines 21--23 are doing, which was stolen directly from Odo's `URL` unittests.

> Could we have the test spin up a local webserver and then serve this test data? I would prefer to be able to run the tests without internet connection.

Yes, I'd prefer that too.  And while we're at it, we should refactor Odo's URL tests to use the same logic.

I don't have the time to do this refactoring this week.  In the interests of time and priorites, I made a separate issue for this improvement (#1360).